sandy/sections/mf2.py

```diff
@@ -182,272 +186,333 @@ def read_mf2(tape, mat):
                 LRU0.update(header2)
                 C, i = sandy.read_cont(df, i)
                 add = {
-                        "SPI": C.C1,        #Spin, I, of the target nucleus.
-                        "AP": C.C2,         #Scattering radius in units of 10e-12cm.
-                        "NLS": C.N1,        #Number of l-values (neutron orbital angular momentum) in this energy region.
-                        }
+                    "SPI": C.C1,  # Spin, I, of the target nucleus.
+                    "AP": C.C2,  # Scattering radius in units of 10e-12cm.
+                    # Number of l-values (neutron orbital angular momentum) in
+                    # this energy region.
+                    "NLS": C.N1,
```

Comment:
I still don't understand if NLS can be not zero for LRU=0.
If not, remove, else, add a test.

Reply:
In principle LRU=0 is identical to the first part of LRU=1. There is no need to code it twice